spec/models/concerns/preservable_spec.rb

```diff
@@ -49,11 +49,18 @@ class PreservableItem < ActiveFedora::Base
 
   end
 
-  it 'builds the technicalMetadata datastream' do
+  it 'builds the technicalMetadata datastream if the object is an item' do
+    allow(item).to receive(:is_a?).and_return(Dor::Item)
```

Comment:
i think `is_a?` is supposed to return a boolean:
http://ruby-doc.org/core-2.4.1/Object.html#method-i-is_a-3F